recipes-qt/maliit/maliit-framework-qt5_git.bb

```diff
@@ -53,6 +53,7 @@ EXTRA_QMAKEVARS_PRE = "\
     CONFIG+=local-install \
     CONFIG+=nosdk \
     CONFIG+=nodoc \
+    CONFIG+=notests \
```

Comment:
I haven't seen them failing with gcc, can we disable them only for clang? Or have you seen failing them like this with gcc?

Reply:
It is indeed a valid point, is it possible to "fix" the tests?

Reply:
lets change it to 

```
EXTRA_QMAKEVARS_PRE_append_toolchain-clang = " notests"
```
for now.